packages/regenesis-surgery/test/eoa.spec.ts

```diff
@@ -62,11 +62,20 @@ describe('EOAs', () => {
     })
   })
 
-  // Does not exist on Kovan?
-  describe.skip('1inch deployer', () => {
+  // eslint-disable-next-line
+  describe('1inch deployer', function() {
     let eoa: Account
-    before(() => {
+    // eslint-disable-next-line
+    before(function() {
```

Comment:
Have you tested this? I think this would need to be an arrow function and the parent should be a "normal" function. Not 100% sure though.

Reply:
I have tested it and it works, it will skip as expected when running on kovan